x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/util/DateUtils.java

```diff
@@ -16,39 +16,88 @@
 import java.time.ZoneId;
 import java.time.ZoneOffset;
 import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatterBuilder;
+import java.util.Locale;
 
-public class DateUtils {
+import static java.time.format.DateTimeFormatter.ISO_LOCAL_DATE;
+
+public final class DateUtils {
+
+   public static final java.time.format.DateTimeFormatter ISO_DATE_ONLY = new DateTimeFormatterBuilder()
```

Comment:
Isn't the one in the JDK enough: https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_LOCAL_DATE ?